Tools/clinic/clinic.py

```diff
@@ -4984,25 +5008,21 @@ def run_clinic(parser: argparse.ArgumentParser, ns: argparse.Namespace) -> None:
             parser.error(
                 "can't specify --converters and a filename at the same time"
             )
-        converters: list[tuple[str, str]] = []
-        return_converters: list[tuple[str, str]] = []
-        ignored = set("""
-            add_c_converter
-            add_c_return_converter
-            add_default_legacy_c_converter
-            add_legacy_c_converter
-            """.strip().split())
-        module = globals()
-        for name in module:
-            for suffix, ids in (
-                ("_return_converter", return_converters),
-                ("_converter", converters),
-            ):
-                if name in ignored:
-                    continue
-                if name.endswith(suffix):
-                    ids.append((name, name.removesuffix(suffix)))
-                    break
+        any_converter_type = ConverterType | ReturnConverterType
+        converter_list: list[tuple[str, any_converter_type]] = []
+        return_converter_list: list[tuple[str, any_converter_type]] = []
```

Comment:
```suggestion
        AnyConverter = ConverterType | ReturnConverterType
        converter_list: list[tuple[str, AnyConverter]] = []
        return_converter_list: list[tuple[str, AnyConverter]] = []
```

Reply:
... or `AnyConverterType`. At least follow the typing style convention :)

Reply:
I changed the name for: AnyConverterType